aws/table_aws_ebs_snapshot.go

```diff
@@ -362,8 +332,9 @@ func buildEbsSnapshotFilter(ctx context.Context, d *plugin.QueryData, h *plugin.
 	if equalQuals["owner_id"] != nil {
 		ownerFilter.Name = aws.String("owner-id")
```

Comment:
When setting the owner ID, I think we should use the `OwnerIds` input param instead of `owner-id` in the `Filters`, as AWS recommends:
>     //    * owner-id - The Amazon Web Services account ID of the owner. We recommend that you use the related parameter instead of this filter.

Reply:
Similarly, we should use `SnapshotIds` instead of `snapshot-id` in `Filters`